### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by Mus156** [entropyxyz/community#38] [Get Test Funds] [answered]
### Address

5Ckj3i79huRmTG1LDUcejAJpCtnnLyRM7eLatgmvwWkK8rvp

**Comment by johnnymatthews** [comment, marked as answer]
Sent, thanks!

**Reply by Mus156** [reply]
I checked my balance in the CLI but it's empty, what could this be related to? Thanks.
![123](https://github.com/entropyxyz/community/assets/92182876/18ed01ae-b9c2-4b0e-aa7e-4b4e9654371c)

**Reply by johnnymatthews** [reply]
Ah, sorry about that. It turns out there is a bug in how the CLI transfers funds -- or rather _doesn't_ complete the request occasionally. This is being fixed.

In the meantime, I've sent you some testnet funds via [another method](https://docs.entropy.xyz/guides/transfer-funds#substrate-explorer). You should have them now.

**Reply by Mus156** [reply]
I hear you, thank you. I have one more question, could I become a validator in your project? And what do I need to do for that.

**Reply by johnnymatthews** [reply]
The project isn't currently ready for external validators. We'll make an annoucement closer to the time though!

**Reply by Mus156** [reply]
I'll be looking forward to this opportunity!
